client/app/domain/domain-tabs.controller.js

```diff
@@ -131,6 +133,12 @@ angular.module('App').controller(
       } else {
         this.selectedTab = this.defaultTab;
       }
+
+      this.atInternet.trackClick({
```

Comment:
Did you check if this work ? Because the expected names are very precise and I think the name of each tab is in UPPERCASE :)

Reply:
yup all tabs are in uppercase, will verify this